.github/CODEOWNERS

```diff
@@ -207,6 +207,7 @@ app/models/accredited_organization.rb @department-of-veterans-affairs/accredited
 app/models/adapters/payment_history_adapter.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
 app/models/appeal_submission.rb @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/appeal_submission_upload.rb @department-of-veterans-affairs/benefits-decision-reviews-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/models @department-of-veterans-affairs/octo-identity
```

Comment:
We def don't want to own the entire `app/models` directory, we should add only the files that are added here